hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -159,11 +159,18 @@ public void writeDbDataToStream(DBCheckpoint checkpoint,
       archiveOutputStream
           .setBigNumberMode(TarArchiveOutputStream.BIGNUMBER_POSIX);
       // Files to be excluded from tarball
-      Set<Path> toExcludeFiles = normalizeExcludeList(toExcludeList,
-          checkpoint.getCheckpointLocation());
+      RocksDBCheckpointDiffer differ =
+          getDbStore().getRocksDBCheckpointDiffer();
+      DirectoryData sstBackupDir = new DirectoryData(tmpdir,
+          differ.getSSTBackupDir());
+      DirectoryData compactionLogDir = new DirectoryData(tmpdir,
+          differ.getCompactionLogDir());
+
+      Map<Path, Path> toExcludeFiles = normalizeExcludeList(toExcludeList,
```

Comment:
Is `toExcludeList` only applicable to sstFiles? or also to compactionLogFiles?
nit: if it is for sstFiles only, may be variable name can be changed `toExcludeSstFileList` to emphasize on SST part.

Reply:
I'll change it to sstFilesToExclude.